src/test/daml/Daml/Finance/Instrument/Equity/Test/Dividend.daml

```diff
@@ -219,17 +224,21 @@ run = script do
   -- stock (1.5 additional shares).
   -- perUnitDistribution is an arbitrary list, so this can be extended with additional options, e.g.
   -- warrants or cash in a different currency.
-  distributionEventCashCid <-
-    Instrument.submitExerciseInterfaceByKeyCmd @Equity.I [issuer] [] cumEquityInstrument
+  (cumEquityInstrumentCid, _) <- queryAnyByKey @Equity.T @BaseInstrument.I [issuer]
+    cumEquityInstrument
+  distributionEventCashCid <- submitMulti [issuer] [] do
```

Comment:
I think there are a few more instances of this kind where we use `submitMulti` unnecessarily